central/sensor/service/pipeline/complianceoperatorresultsv2/pipeline.go

```diff
@@ -0,0 +1,83 @@
+package complianceoperatorresultsv2
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	v2 "github.com/stackrox/rox/central/complianceoperator/v2/checkresults/datastore"
+	"github.com/stackrox/rox/central/convert/internaltov2storage"
+	countMetrics "github.com/stackrox/rox/central/metrics"
+	"github.com/stackrox/rox/central/sensor/service/common"
+	"github.com/stackrox/rox/central/sensor/service/pipeline"
+	"github.com/stackrox/rox/central/sensor/service/pipeline/reconciliation"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/metrics"
+)
+
+var (
+	_ pipeline.Fragment = (*pipelineImpl)(nil)
+)
+
+// GetPipeline returns an instantiation of this particular pipeline
+func GetPipeline() pipeline.Fragment {
+	return NewPipeline(v2.Singleton())
+}
+
+// NewPipeline returns a new instance of Pipeline.
+func NewPipeline(v2Datastore v2.DataStore) pipeline.Fragment {
+	return &pipelineImpl{
+		v2Datastore: v2Datastore,
+	}
+}
+
+type pipelineImpl struct {
+	v2Datastore v2.DataStore
+}
+
+func (s *pipelineImpl) Capabilities() []centralsensor.CentralCapability {
+	return nil
+}
+
+func (s *pipelineImpl) Reconcile(_ context.Context, _ string, _ *reconciliation.StoreMap) error {
+	// Not currently deleting anything for historical purposes
```

Comment:
Can you elaborate that this is for the purposes of keeping history. I read this as "historical reason" which just generally means because of past precedent as opposed to the goal of doing trending / historical look back